src/net.c

```diff
@@ -16,6 +21,8 @@
 
 #define UEVENT_BUFFER_SIZE 512
 
+#define MAC_ADDR_STR_SIZE 18
```

Comment:
Could do with a comment showing format `XX:XX:XX:XX:XX:XX + 1` maybe. I'd consider naming it `IPV4_MAC_ADDR_STR_SIZE` to make it clearer that this is only for IPv4.